spring-boot-parent/src/checkstyle/checkstyle-header.txt

```diff
@@ -1,5 +1,5 @@
 ^\Q/*\E$
-^\Q * Copyright \E20\d\d\-20\d\d\Q the original author or authors.\E$
+^\Q * Copyright \E20\d\d(\-20\d\d)?\Q the original author or authors.\E$
```

Comment:
IANAL, but we've gone with 2012-201n for the rest of Boot's code base so I think we should be consistent with that for now.